scipio/src/executor.rs

```diff
@@ -941,9 +940,9 @@ impl<T> Task<T> {
     ///
     /// # Examples
     /// ```
-    /// use scipio::{LocalExecutor, Local, Latency};
+    /// use scipio::{LocalExecutor, Local, Latency, LocalExecutorBuilder};
     ///
-    /// let ex = LocalExecutor::spawn_executor("example", None, || async move {
+    /// let ex = LocalExecutorBuilder::new().spawn_thread("example", || async move {
```

Comment:
if we have a builder API, then I think we should pass the name to it as well. That would allow us to create unnamed executors, which is fine for many applications, and simpler.